frontend/src/components/home/index.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
-import { Route, Redirect, BrowserRouter as Router } from 'react-router-dom';
+import { Route, Redirect } from 'react-router-dom';
 import Sidebar from '../sidebar';
 import Employees from '../employees';
 import Rota from '../rota';
 import Summary from '../summary';
 import homeStyles from './home.module.scss';
 
 const Home = () => (
-  <Router>
+  <>
```

Comment:
I've been urged to avoid creating containers using an empty tag. I cannot find much info about why you should or shouldn't at the moment, but I'd probably just make this a div with a, informative className so we can identify it later on

Reply:
I could potentially move the Router back here?